app/views/helpers/TimelineEvents.scala

```diff
@@ -43,9 +43,8 @@ class TimelineEvents @Inject() (
   def apply(submission: Submission)(implicit messages: Messages): Seq[TimelineEvent] = {
 
     val notificationSummaries = submission.actions.flatMap { action =>
-      val summaries = action.notifications.fold(Seq.empty[NotificationSummary])(_.filterNot(_.enhancedStatus == CUSTOMS_POSITION_GRANTED))
-      if (action.requestType != CancellationRequest) summaries
-      else summaries ++ Seq(NotificationSummary(UUID.randomUUID, action.requestTimestamp, REQUESTED_CANCELLATION))
+      if (action.requestType != CancellationRequest) action.notifications.fold(Seq.empty[NotificationSummary])(identity)
+      else Seq(NotificationSummary(UUID.randomUUID, action.requestTimestamp, REQUESTED_CANCELLATION))
```

Comment:
This does not look like it will include the `CUSTOMS_POSITION_DENIED` notification that is sent against the `CancellationRequest` :thinking:

Reply:
Won't `CUSTOMS_POSITION_DENIED` be received  against a `SubmissionRequest`? :thinking:

Reply:
My bad. Don't know why... I was convinced `CUSTOMS_POSITION_DENIED` is received against a `SubmissionRequest.